src/App.css

```diff
@@ -18,7 +18,64 @@
   font-size: large;
 }
 
+#text-input {
+  margin-top: 100px;
+}
+
 @keyframes App-logo-spin {
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+ul {
+  list-style: none;
+}
+
+#shopping-cart-logo {
```

Comment:
Agreed! Classes are great because they're reusable all over the app, so we should use them as much as possible. IDs are good for any one-offs, but they're tricky to overwrite.